pkg/scheduler/core/spreadconstraint/group_clusters.go

```diff
@@ -117,7 +117,7 @@ func (info *GroupClustersInfo) generateClustersInfo(clustersScore framework.Clus
 	clustersReplicas := calAvailableReplicas(clusters, rbSpec)
 	for i, clustersReplica := range clustersReplicas {
 		info.Clusters[i].AvailableReplicas = int64(clustersReplica.Replicas)
-		info.Clusters[i].AvailableReplicas += int64(getScheduledReplicas(rbSpec, clustersReplica.Name))
+		info.Clusters[i].AvailableReplicas += int64(rbSpec.AssignedReplicasForCluster(clustersReplica.Name))
```

Comment:
@huone1 Can we be sure that `rbSpec` is not nil here? Otherwise, it will panic.

Reply:
The failing unit test is due to `nil` `speck` at :
https://github.com/karmada-io/karmada/blob/28a6c2c7672a5efd504c9c264bf2c6f53e5e89b4/pkg/scheduler/core/spreadconstraint/group_clusters_test.go#L54-L57

Reply:
It will not be a null pointer  because the spec is structure ， not a pointer in the definition of resource binding API